Tests/LibWeb/WPT/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+if [ -z "$SERENITY_SOURCE_DIR" ]
+then
+    echo "SERENITY_SOURCE_DIR is not set. Exiting."
+fi
+
+# NOTE: WPT runner assumes Ladybird, WebContent and WebDriver are available in $PATH.
+export PATH="${SERENITY_SOURCE_DIR}/Build/lagom/bin:${SERENITY_SOURCE_DIR}/Meta/Lagom/Build/bin:${PATH}"
+
+# Install dependencies.
+sudo apt-get install -y git python3 python3-pip python3-venv libssl-dev
+
+# Ensure a `python` binary exists
+sudo apt-get install -y python-is-python3
+
+# Clone patched web-platform-tests repository
+git clone --depth 10000 https://github.com/web-platform-tests/wpt.git
+
+# Switch to the commit that was used to generate tests expectations. Requires periodic updates.
+(cd wpt; git checkout 4c27189ed2db4ddad8e727d4ea9ae8329c3e1672)
```

Comment:
This seems a bit weird. If we clone with depth 10000, won't 4c27189ed2db4ddad8e727d4ea9ae8329c3e1672 eventually become unavailable if they keep adding commits?

Either way, I guess if we keep updating the hash, it's fine for now..